Objective-C/syntax_test_objc.m

```diff
@@ -581,6 +581,9 @@ @interface Person : NSObject
 /* <- keyword.other punctuation.definition.keyword */
 /*^ keyword.other */
 @property (readonly) NSString *lastName;
+/* <- keyword.other punctuation.definition.keyword */
+/*^ keyword.other */
+@property (readonly, nullable, class) NSString *middleName;
 @end
```

Comment:
Some assertions should be added here to make sure we are testing this change. Right now I could back out the change to the .sublime-syntax and I'm fairly sure the tests would still all pass.